plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -53,6 +53,7 @@ struct GsPluginData
 	char *personality;
 	char *product_name;
 	gboolean eos_arch_is_arm;
+	FlatpakInstallation *inst;
```

Comment:
Nitpick: I’d recommend calling the member `installation` rather than `inst`. It’s not much more typing, and `inst` could be misinterpreted as a contraction of `instance`.